Update/kage_005.txt

```diff
@@ -2167,7 +2167,7 @@ void main()
 //rやはり彼も一連の事件で危機感、そして焦燥感を抱いていたために、その苛立ちからつい強引に急いて、仕損じてしまったということだろう。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "やはり彼も一連の事件で危機感、そして焦燥感を抱いていたために、その苛立ちからつい強引に急いて、仕損じてしまったということだろう。",
-			NULL, "After all, he recognized the crisis behind this series of incidents. The more that frustrated him, the more he wanted to hurry and force his frustration away, and that led to this mistake.", Line_Normal);
+			NULL, "He too could tell just dangerous these series of incidents were, and because of his impatience, that irritation drove him to be pushy, which led to this mistake.", Line_Normal);
```

Comment:
```suggestion
			NULL, "He too could tell just how dangerous these series of incidents were, and because of his impatience that irritation drove him to be aggressive, which led to this mistake.", Line_Normal);
```